data/campaigns/Under_the_Burning_Suns/scenarios/03_Stirring_in_the_Night.cfg

```diff
@@ -657,6 +657,16 @@
         [/message]
         [message]
             speaker=Garak
+            # po: Garak dies in this scenario, and the important part of this conversation is to tell the player that.
+            #
+            # The English text is currently wrong, it's meant to say "dreams this night" not "dreams of this night", a
+            # typo that will be corrected in 1.18 but not 1.16. The next paragraph is talking about what the text should
```

Comment:
Why is this not simply fixed?

Reply:
I'm wanting to forward-port first, so that there's a commit on the master branch that matches the 1.16 branch.

Reply:
The question about it is still open as well, #7634.